OpenTrons/opentrons-api
# feat(app): add Fixture table to Modules & deck screen under protocol setup

# Overview
Add Fixture table without status. The status will be implemented by RAUT-670

![Screenshot 2023-10-11 at 3 03 30 PM](https://github.com/Opentrons/opentrons/assets/474225/d5f3d833-8267-445c-8884-b4e340f78c26)


close RAUT-668


# Test Plan
0. push this branch to a dev kit or Flex
1. Set up a protocol
2. Tap `Modules & deck`
Check the Fixture table is displayed under the Modules table


# Changelog
- create FixtureTable component and add the very basic test
- update the text in ProtocolSetup screen Modules -> Modules & deck
- update tests for the above changes



# Review requests



# Risk assessment
low
